.circleci/config.yml

```diff
@@ -102,6 +102,9 @@ jobs:
           command: |
             sudo npm i -g serverless
             yarn install
+      - run:
+          name: Add .env file (react-native-dotenv)
```

Comment:
Can you please just remove that `(react-native-dotenv)` it is not relevant here. We can keep just `Add .env file`

Reply:
sure!